cypress/support/step-definitions/select-steps.js

```diff
@@ -92,3 +92,26 @@ When('Type {string} text into input and select the value', (text) => {
 When('Type {string} text into input and select the value into iFrame', (text) => {
   selectInputNoIframe().type(`${text}{downarrow}{enter}`);
 });
+
+Then('Select is transparent', () => {
+  select().parent().should('have.css', 'background-color', 'rgba(0, 0, 0, 0)')
+    .and('have.css', 'border-left-color', 'rgba(0, 0, 0, 0.85)')
```

Comment:
could we make a `const color = rgba(0, 0, 0, 0.85)` and use it in every `.and` ?

Reply:
fixed